egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -194,6 +221,23 @@ def PrintConfig(file_name, config_lines):
     f.write("\n".join(config_lines['component-nodes']))
     f.close()
 
+def ParseCnnString(cnn_string):
```

Comment:
So the user can input the parameters as shown in the param_string and you could parse them using argparse as above